rpms/microshift.yml

```diff
@@ -11,6 +11,10 @@ content:
       url: git@github.com:openshift/microshift.git
     specfile: packaging/rpm/microshift.spec
     modifications:
+    - action: replace
+      match: "# BuildRequires: golang >= %{golang_version} Temporarily disabled until go1.18 is available in RHEL8 repositories"
```

Comment:
I think we're good to remove this after https://github.com/openshift/microshift/pull/1059/files ?

Reply:
Yes, I've removed the workaround.